code-challenges.js

```diff
@@ -9,40 +9,81 @@
 
 // Create the code and test each of the variables provided. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
+//I need to use 'if' and 'else if' statements to set a range for the numbers to output desired. I will have to set 3 outputs based around 212 . One at temp, one below and one above. I will use template literals to console back the exact temperature that was entered.
```

Comment:
well done! I remember you mentioning struggling with pseudo code and this is great!